core/templates/dev/head/components/OutcomeDestinationEditorDirective.js

```diff
@@ -55,7 +56,7 @@ oppia.directive('outcomeDestinationEditor', [
           // development.
           $scope.canEditRefresherExplorationId = (
             GLOBALS.isAdmin || GLOBALS.isModerator);
-          $scope.explorationIdPattern = /^[a-zA-Z0-9_-]+$/;
+          $scope.globalIdPattern = /^[a-zA-Z0-9_-]+$/;
```

Comment:
Maybe let's call this explorationAndSkillIdPattern. I think globalIdPattern is a bit _too_ generic.

Better still, make this into an oppia.constant.

Reply:
Done.